staging/src/k8s.io/kubectl/pkg/cmd/get/get.go

```diff
@@ -266,6 +269,9 @@ func (o *GetOptions) Complete(f cmdutil.Factory, cmd *cobra.Command, args []stri
 		if o.ServerPrint {
 			printer = &TablePrinter{Delegate: printer}
 		}
+		if !o.ShowManagedFields {
```

Comment:
Why in `Get`? I think you can wired it up into https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/cli-runtime/pkg/genericclioptions/json_yaml_flags.go#L45-L59, and the flag can be added here: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/cli-runtime/pkg/genericclioptions/json_yaml_flags.go#L63. I think it's better if the option is applied to all commands that print objects!

Unfortunately `kubectl diff` is a little different, we could take care of that right after.

Reply:
Updated as you said.